app/assets/stylesheets/components/page_specific/_floating_chat.scss

```diff
@@ -1,11 +1,11 @@
 .chat-box__container,
-.chat-popup__container {
+.mobile-webchat__container {
   background-color: $color-white;
   position: fixed;
-  z-index:99;
-  -webkit-box-shadow:  0 0 7px 1px rgba(133,133,133,0.71);
-  -moz-box-shadow:  0 0 7px 1px rgba(133,133,133,0.71);
-  box-shadow:  0 0 7px 1px rgba(133,133,133,0.71);
+  z-index: 90;
```

Comment:
why has this been decreased?

Reply:
I have lowered this value on the container because it was conflicting with the X button that shows when the popup is opened

Reply:
We need to look into the z-index value a bit more because right now this is in front of the menu and the overlay that greys out the main page content when the menu is open. We may need to change those values as well to make everything work well together.

Reply:
I spoke with John Player and the popup will need to show behind the menu and the grey overlay every time so I'll be looking to make this change in the next commit